### DISCUSSION TITLE
other method in paper

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#103] [General]
other method in paper whether use the same four training dataset?
is it fair?

**Comment by ZhiyuanChen** [comment]
> other method in paper whether use the same four training dataset?
> is it fair?

Sorry but I am having troubles to understand your question. 
You may use Chinese or other languages if you find it's easier.

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing it for now.
Please reopen it or create a new issue if you have further question.
